src/pages/Login.jsx

```diff
@@ -52,11 +56,11 @@ const LoginScreen = () => {
         });
       })
       .catch((error) =>
-        enqueueSnackbar(error.message, {
+        enqueueSnackbar("Something went wrong. Make sure you already have an account linked with this email-id!", {
```

Comment:
We can't hardcode this error message, someone without an account can also directly login using google sign in.(handled by firebase)

Reply:
So, should I keep it as it was earlier?

Reply:
> So, should I keep it as it was earlier?

You can go through the Firebase documentation to see what actual errors this method raise and decide